infra/ci/build.py

```diff
@@ -213,6 +213,7 @@ def build_base_images():
   images = [
       'base-image',
       'base-builder',
+      'base-builder-new',
```

Comment:
Umm...once we split up the images, we may need to make this function smarter so that it rebuilds base-builder-swift if swift projects are used, base-builder-java if java projecs are used, etc.